server/routers/face/face_router.py

```
@@ -10,149 +12,137 @@
-        faces = cursor.fetchall()
-        return [
-            {
-                "id": face[0],
-                "photo_id": face[1],
-                "face_id": face[2],
-                "album_id": face[3],
-            }
-            for face in faces
-        ]
+async def get_faces(
+    current_user=Depends(get_current_user), session: Session = Depends(get_session)
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)